src/vellum/workflows/state/context.py

```diff
@@ -3,15 +3,21 @@
 from typing import TYPE_CHECKING, Optional
 
 from vellum import Vellum
+from vellum.workflows.events.types import ParentContext
 from vellum.workflows.vellum_client import create_vellum_client
 
 if TYPE_CHECKING:
     from vellum.workflows.events.workflow import WorkflowEvent
 
 
 class WorkflowContext:
-    def __init__(self, _vellum_client: Optional[Vellum] = None):
+    def __init__(
+        self,
+        _vellum_client: Optional[Vellum] = None,
+        _parent_context: Optional["ParentContext"] = None,
```

Comment:
including parent context as WorkflowContext passthrough

Reply:
[nit] don't need the `"quotes"` here if the import above is normal and outside of TYPE_CHECKING